src/styles/container.css

```diff
@@ -0,0 +1,6 @@
+/* TODO: Add max-widths for each breakpoint + paddings for the resolution when the container fluid */
+@layer components {
+  .container {
```

Comment:
Let's talk more about this. It would be good to have an example here so you can just uncomment the code and change numbers.